src/test/scala/chiselTests/util/experimental/PlaSpec.scala

```diff
@@ -49,6 +49,21 @@ class PlaSpec extends ChiselFlatSpec {
     })
   }
 
+  "#2112" should "be generated correctly" in {
+    assertTesterPasses(new BasicTester {
+      val table = Seq(
+        (BitPat("b000"), BitPat("b?01")),
+        (BitPat("b111"), BitPat("b?01")),
```

Comment:
```suggestion
        (BitPat("b111"), BitPat("b?01"))
```
Scala 2.11 does not support trailing commas